os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_central/ble_tizenrt_client.c

```diff
@@ -74,6 +77,167 @@ trble_result_e rtw_ble_client_init(trble_client_init_config* init_parm)
     return TRBLE_SUCCESS; 
 }
 
+
+trble_result_e rtw_ble_client_scan_whiltelist_add(trble_addr *addr)
+{
+    if (client_init_parm == NULL || client_init_parm->trble_device_scanned_cb == NULL || addr == NULL)
```

Comment:
Does addr->type also need a NULL check ?

Reply:
Nope, addr->type is not a pointer, it is trble_addr_type_e which is an enum.